app/views/modals/_manage_group.slim

```diff
@@ -67,4 +67,6 @@ div.modal.fade id="manageGroup" aria-labelledby="manageGroupHeader" aria-describ
           div.modal-footer
             div
               button.btn.btn-danger type="button" data-dismiss="modal" Cancel
+              -if @group.notebooks.count == 0
+                button.btn.btn-danger id='DeleteGroup' type="button" Delete Group
```

Comment:
Are we okay with making the id="deleteGroup"? Double quotes and camel case of the ID with first letter being lowercase to match the rest? I know we'd need to change the JS line for it too.

Reply:
yeah yeah